server/social.coffee

```diff
@@ -221,10 +226,10 @@ module.exports = exports = (log, loga, argv) ->
                 displayName: extractUserInfo(argv.oauth2_DisplayNameField, 'params.user_id')
               }
             }
+            cb(null, user)
```

Comment:
This here is the function call that puts the user on `req.session.passport.user`, where it's later showing up undefined. The change here is a little sus, in that cb is now only getting called in the try block, and will never call back on error. The old way is a little odd too in that it could add a partial user object w/o all of the expected information. This should probably call `cb(e)` in the catch block.

It seems likely that what's happening is that something is throwing an error in here and the feedback isn't getting to the client, is there any info in the error output from the server?